### PR TITLE
+ Returns Data from Mapbox Geocode API given a search parameter

### PR DESCRIPTION
**Pull request by ghost**
Calling geocode with a search term will return a dataset of features from MapBox API. 
Includes Latitude and Longitude.

### CODE REVIEW

**Comment by lukem-kainos** [comment]
+1

**Comment by timja** [comment]
Is it possible to load the access_token from a config file and have that config file git ignored?

File: map-demo/app/assets/javascripts/map.js [hunk 1]
```diff
@@ -7,6 +7,12 @@ L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={
   accessToken: 'pk.eyJ1IjoiYXNobGVpZ2hobyIsImEiOiJjaXN6d3l6MjUwMDYxMnlvM2djajZudG1zIn0.hAdrsz6R7-4Z9OepbVmMOA'
```
Comment (timja) [inline map-demo/app/assets/javascripts/map.js:7]: Err

**Comment by sdonaghey03** [comment]
Was able to break the search, typed in my home postcode 'BT476FE' below returned error.
screen shot 2016-09-14 at 10 31 31

**Comment by timja** [comment]
Can we make the search button look a bit nicer? Maybe some spacing between the input and the button?

File: map-demo/app/views/map.scala.html [hunk 2]
```diff
@@ -12,12 +12,22 @@
                 <strong class="phase-tag">BETA</strong>
                 <span>This is a new service – your <a href="#">feedback</a> will help us to improve it.</span>
             </p>
+            <div id="location-input">
+                <label class="form-label" for="location-search" id="location-text-field">
+                    Search for Location
+                </label>
+                <input class="form-control" id="location-search" type="text">
+                <a class="button button-start" id="search-button" href="#" role="button" onclick="geocode(document.getElementById('location-search').value)">Search</a>
+            </div>
```
Comment (sdonaghey03) [inline map-demo/app/views/map.scala.html]: Can this be moved out of the phase-banner div? Maybe in to it's own seperate div section.
The search button is a little large, drop the 'button-start' class.

Some spacing between header and map would be nice!

**Comment by sdonaghey03** [comment]
👍  but Ash's branch is within this? His branch hasn't been merged yet

**Pull request closed without merging**